service/com.observiq.collector.plist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>com.observiq.collector</string>
+    <key>EnvironmentVariables</key>
+    <dict>
+    <key>OIQ_OTEL_COLLECTOR_HOME</key>
+    <string>[INSTALLDIR]</string>
+    </dict>
+    <key>ProgramArguments</key>
+    <array>
+    <string>[INSTALLDIR]observiq-otel-collector</string>
+    <string>--config</string>
+    <string>[INSTALLDIR]config.yaml</string>
+    <string>--logging</string>
+    <string>[INSTALLDIR]logging.yaml</string>
```

Comment:
We can probably add the flag for manager.yaml. If the file is missing it goes into standalone mode.

Reply:
Good catch, I copy pasted this from the old service definition we had (before goreleaser added options for this).

I added in KeepAlive and WorkingDirectory, as well as the manager.yaml command line.